libarchive/Makefile

```diff
@@ -1,19 +1,24 @@
-CFLAGS     = -g
-INCLUDE    = -I/usr/include -I.
-LIBS       = -larchive
+CFLAGS     = -g 
+INCLUDE    = -I/usr/local/include -I/usr/include -I.
+LIBS       = -L /usr/local/lib -larchive
 
-PYVER ?= 2.7
+#DEFINES = -DPyUnicode_AS_STRING=PyUnicode_AS_DATA
+PYVER ?= 3.9
 
 all: __libarchive.so
 
 _libarchive_wrap.c: _libarchive.i
-	swig -python -shadow _libarchive.i
+	#swig -python -c++ -Wall -shadow _libarchive.i
```

Comment:
Can the commented out lines in this file be removed?

Reply:
Done